### ISSUE TITLE
Retrieve headers from response?

### ISSUE THREAD

**Issue by jckspencer** [Altran-PT-GDC/Robot-Framework-SOAP-Library#43]
I need to retrieve the headers from a response in order to get cookie information to pass along in another request. From what I can tell, the zeep response includes this information but all of the Call SOAP Method keywords strip away the headers and return only the xml text response. Might there be a way to preserve the response headers and pass them along into robot? RequestsLibrary does this with the dot operator, however if this were introduced in SoapLibrary, I imagine a lot of existing code would break. I'm brainstorming ways to do this but wanted to open an issue in the meantime in case anyone has ideas off the bat.

**Comment by samuelpcabral** [comment]
After some tests, I think the best way is get the entire response object and run something like this:

```
Test Get Last Response Object
    Create Soap Client    ${wsdl}    ssl_verify=False
    Call SOAP Method With XML    request.xml
    ${response_object}=    Get Last Response Object
    ${response_header}=    Set Variable    ${response_object.headers}
```

You can take a look and brainstorm some ideas in this branch: https://github.com/Altran-PT-GDC/Robot-Framework-SOAP-Library/tree/response_header

**Comment by jckspencer** [comment]
> @jckspencer Hi, when the library was created, the goal was to make it as simple as possible, as when we use SoapUI, we send a request and receive the response... so it only returns the etree object from the xml of the response. (maybe it was a mistake, but now I wouldn't like to change the library making it incompatible with previous versions.)
> 
> a simpler solution instead of changing the existing keywords, would be to create something more direct like "get header from last response" and call it right after the request.
> 
> or it could be something more generic like: "get last response object" and the user could get all the attributes of the response object that zeep creates. through the dot operator

Completely agree with the intentions of keeping the library simple. I don't feel that the original implementation is a mistake, just that there is room for more functionality that could benefit users like me. The idea of a Get Last Response Object keyword was my thought for this as well and I think is the best solution. I've looked in the branch and your implementation is exactly how I would have handled it.

**Comment by samuelpcabral** [comment]
Released in the new version 1.3 `pip install --upgrade robotframework-soaplibrary`
